oclint-scripts/make

```diff
@@ -1,4 +1,4 @@
 #! /bin/sh -e
 
-./ci -reset
-./ci -setup -release
+./ci -reset "$@"
```

Comment:
`-reset` does bunch of deletes, not sure if we really need it to run concurrently.